Assets/MRTK/Providers/LeapMotion/Serialized/Microsoft.MixedReality.Toolkit.Providers.LeapMotion.Serialized.asmdef

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Microsoft.MixedReality.Toolkit.Providers.LeapMotion.Serialized",
+    "references": [
+        "Microsoft.MixedReality.Toolkit"
+    ],
+    "includePlatforms": [],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": false,
+    "precompiledReferences": [],
+    "autoReferenced": true,
+    "defineConstraints": [],
+    "versionDefines": [],
+    "noEngineReferences": false
+}
```

Comment:
@keveleigh mentioned `versionDefines` and `noEngineReferences` are new to Unity 2019.  Some of the asmdefs have these fields while others do not.  Do we want all of the asmdefs to align with Unity 2018 or 2019?